kaggle_environments/envs/lux_ai_2021/lux_ai_2021.json

```diff
@@ -29,7 +33,7 @@
       "type": "boolean",
       "default": false
     },
-    "actTimeout": 1,
+    "actTimeout": 3,
```

Comment:
is this doable? @bovard 

This matches the same config as last year, not sure why I initially copied over 1s

Reply:
yup!